app/components/NextEventInfo/NextEventInfo.tsx

```diff
@@ -0,0 +1,57 @@
+import type { MeetupEvent } from "~/models/meetup.parsing";
+import { isEmptyString } from "~/utils";
+import MeetupLink from "~/components/MeetupLink";
+import type { SerializeFrom } from "@remix-run/server-runtime";
+
+const DEFAULT_VENUE: MeetupEvent["venue"] = {
+  name: "H-E-B Digital & Favor Eastside Tech Hub (and online)",
+  address: "2416 E 6th St",
+  city: "Austin",
+  state: "TX",
+};
+
+const EVENT_TIME_FORMAT = new Intl.DateTimeFormat("en-US", {
+  weekday: "short",
+  month: "short",
+  day: "numeric",
+  hour: "numeric",
+  minute: "2-digit",
+  timeZoneName: "short",
+  timeZone: "America/Chicago",
+});
+
+function isUndefinedVenue(venue: MeetupEvent["venue"]): boolean {
+  return (
+    isEmptyString(venue.address) ||
+    isEmptyString(venue.city) ||
+    isEmptyString(venue.state)
+  );
+}
```

Comment:
I don't think the logic of `isEmptyString` is quite right, because `isUndefinedVenue({})` will return `false`, when it should return `true`.

Might want to add a test to `NextEventInfo.test.tsx` to check for missing values, not just values with empty strings

Reply:
interesting. `{}` isn't a valid scenario due to the yup parsing. in order to build a piece of data to test, you'd have to force a cast just to build it. thus it feels a bit overkill to handle that scenario.